library/src/main/java/com/google/android/exoplayer2/text/ImageCue.java

```diff
@@ -0,0 +1,41 @@
+package com.google.android.exoplayer2.text;
+
+import android.graphics.Bitmap;
+
+public class ImageCue extends Cue {
+
+  final private long start_display_time;
+  final private int x;
```

Comment:
A lot of these parameters appear to be duplicating what's already in the Cue base class. Why can't the cue be sized using the existing fields (line, position, size) with (lineType=LINE_TYPE_FRACTION). It feels a lot like this should be supported just by adding a Bitmap field directly into Cue, that can be set instead of (text, textAlignment).

A few asides:
- It's unclear what units these variables are in.
- We tend to just make final immutable fields public, rather than having getter methods.
- "final private" -> "private final" (or "public final" as per comment above) as per the [Google style guide](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers).